### DISCUSSION TITLE
[Question] Is this project still alive?

### DISCUSSION THREAD

**Discussion by accetto** [niieani/bash-oo-framework#90] [General]
Hello, 

is this project still maintained? If yes, then by whom?

Regards,
accetto

**Comment by niieani** [comment]
Hi Accetto,

I don't have enough time for active development of v3 at this time, however I still am able to review and merge fixes / new functionality submitted by the community.

I'm assuming it's probably the same for @nkakouros.

**Comment by nkakouros** [comment]
Yes, it is the same here. I would love to work on this though when the time permits (summer?).

**Comment by adudek** [comment]
I would love to use it (and therefore support) in proffessional setting. Once it will get workable list/array support I'm first one to dive-in.

**Comment by niieani** [comment]
Just FYI I've also been toying around with the idea of a typed language compiled to bash - [bashscript](https://github.com/niieani/bashscript).

It's definitely a different approach and not meant to replace this project. But still think it has potential. Though things like objects, references/pointers and garbage collection are hard complicated to model in bash, but I've jotted down ideas for some of these in the readme (not implemented).

**Comment by niieani** [comment]
Also, enabled [GitHub discussions](https://github.com/niieani/bash-oo-framework/discussions) for this repo, so we can continue over there.

**Comment by accetto** [comment]
Thanks a lot for all the answers. I'm glad that it's not over yet. I've been testing it last few days and I think I'll probably send one or two PRs for some smaller stuff. Overall it looks promising and I'll continue with evaluation. Nice work. :)
